app/controllers/podcasts_controller.rb

```diff
@@ -0,0 +1,74 @@
+class PodcastsController < ApplicationController
+  before_action :set_podcast, only: [:show, :edit, :update, :destroy]
+
+  # GET /podcasts
+  # GET /podcasts.json
+  def index
+    @podcasts = Podcast.all
+  end
+
+  # GET /podcasts/1
+  # GET /podcasts/1.json
+  def show
+  end
+
+  # GET /podcasts/new
+  def new
+    @podcast = Podcast.new
+  end
+
+  # GET /podcasts/1/edit
+  def edit
+  end
+
+  # POST /podcasts
+  # POST /podcasts.json
+  def create
+    @podcast = Podcast.new(podcast_params)
+
+    respond_to do |format|
+      if @podcast.save
+        format.html { redirect_to @podcast, notice: 'Podcast was successfully created.' }
+        format.json { render :show, status: :created, location: @podcast }
+      else
+        format.html { render :new }
+        format.json { render json: @podcast.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /podcasts/1
+  # PATCH/PUT /podcasts/1.json
+  def update
+    respond_to do |format|
+      if @podcast.update(podcast_params)
+        format.html { redirect_to @podcast, notice: 'Podcast was successfully updated.' }
+        format.json { render :show, status: :ok, location: @podcast }
+      else
+        format.html { render :edit }
+        format.json { render json: @podcast.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /podcasts/1
+  # DELETE /podcasts/1.json
+  def destroy
+    @podcast.destroy
+    respond_to do |format|
+      format.html { redirect_to podcasts_url, notice: 'Podcast was successfully destroyed.' }
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.